modules/custom/activity_send/modules/activity_send_email/src/Plugin/EmailFrequency/Immediately.php

```diff
@@ -25,15 +24,10 @@ class Immediately extends EmailFrequencyBase {
   /**
    * {@inheritdoc}
    */
-  public function processItem(Activity $activity, Message $message, User $target) {
-    if ($target->isBlocked()) {
-      return;
-    }
-
+  public function processItem(Activity $activity, $body_text, User $target) {
     // Continue if we have text to send and the user is currently offline.
     if (isset($activity->field_activity_output_text) && EmailActivityDestination::isUserOffline($target)) {
       $langcode = $target->getPreferredLangcode();
-      $body_text = EmailActivityDestination::getSendEmailOutputText($message, $langcode);
```

Comment:
This is where we can say:

``` if (!$body_text) {
      $body_text = EmailActivityDestination::getSendEmailOutputText($message, $langcode);
      }
      ```
      So by default for existing Queue Items $body_text will be NULL, but for those who have it passed along to the queue item, we can skip the render fase.
      
      That makes it backwards compatible :)